easybuild/main.py

```diff
@@ -579,9 +580,11 @@ def process_eb_args(eb_args, eb_go, cfg_settings, modtool, testing, init_session
     return overall_success
 
 
-def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
+def main(eb_go=None, cfg_settings=None, args=None, logfile=None, do_build=None, testing=False, modtool=None):
```

Comment:
better add these at the end, to avoid breaking existing uses of `main` like `main(args)`